pimd/pim_igmp.c

```diff
@@ -50,13 +51,12 @@ static int igmp_sock_open(struct in_addr ifaddr, struct interface *ifp,
 {
 	int fd;
 	int join = 0;
-	struct in_addr group;
+	pim_addr group;
 
 	fd = pim_socket_mcast(IPPROTO_IGMP, ifaddr, ifp, 1);
 
 	if (fd < 0)
 		return -1;
-
```

Comment:
Please remove the whitespace changes (multiple locations)

Reply:
Ack